tests/ci/fast_test_check.py

```diff
@@ -90,9 +92,15 @@ def main():
 
     repo_path = Path(REPO_COPY)
 
+    build_output_temp_path = repo_path / "ci" / "tmp"
```

Comment:
It's the same as `TEMP_PATH` by default, you can omit it

Reply:
No, it is not

- `build_output_temp_path` - `/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/ci/tmp`
- `TEMP_PATH` - `/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/**tests**/ci/tmp`

And this, is the difference

Reply:
And if we will use the same, then we need to distiguish those two, which one was already uploaded on the previous step (either by deleting or moving, or splitting into different folders)

Reply:
Let's continue here - https://github.com/ClickHouse/ClickHouse/pull/76416